includes/admin/meta-boxes/views/html-related-orders-row.php

```diff
@@ -11,52 +11,52 @@
 ?>
 <tr>
 	<td>
-		<a href="<?php echo esc_url( get_edit_post_link( $order->get_id() ) ); ?>">
+		<a href="<?php echo esc_url( get_edit_post_link( $row_order->get_id() ) ); ?>">
```

Comment:
Changed to `$row_order` to remove confusion between the global/current order and related orders in each row.